Jenkinsfile-e2e

```diff
@@ -102,7 +104,7 @@ node {
                   colour: 'good',
                   emoji: '🤟'
                 ]
-                notifySlack(messageParameters)
+                // notifySlack(messageParameters, '#si_repo-simorgh')
```

Comment:
why is this commented out?

Reply:
> Continue posting in the si_repo-simorgh channel only if any pipeline step fails.
It says it on the issue. I haven't finished yet tho

Reply:
we should probably treat. unstable and aborted builds as failures too, so will be good to send to the alarms channel